packages/main/src/MenuItem.js

```diff
@@ -56,7 +56,19 @@ const metadata = {
 		},
 
 		/**
-		 * Indicates if the any of the element siblings have children items.
+		 * Defines the accessible ARIA name of the component.
+		 *
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 * @since 1.6.0
```

Comment:
1.7.0

Reply:
We are currently at 1.6.0, so this will go in 1.7.0

Reply:
Done